packages/cozy-harvest-lib/src/components/AccountsList/AccountsListItem.jsx

```diff
@@ -14,7 +14,7 @@ export class AccountsListItem extends React.PureComponent {
 
     const accountName = Account.getAccountName(account)
     const accountLogin = Account.getAccountLogin(account)
-    const nameAndLoginDiffer = accountName !== accountLogin
+    const nameAndLoginDiffer = accountLogin && accountName !== accountLogin
```

Comment:
I think this was the original code, and I removed the first condition because it didn't seem to make any sense. Can we rename the variable to something like `shouldShowAccountLogin` to show that it's not just about the difference ?